src/components/field-types/SingleSelectFieldComponent.vue

```diff
@@ -5,9 +5,7 @@
       <label :for="field.id">{{ field.label }}</label>
 
       <div class="input-group">
-
         <v-select v-model="localValue"
-                  class="form-control"
```

Comment:
That has nasty side-effects:
![Schermafbeelding 2021-04-29 om 17 15 30](https://user-images.githubusercontent.com/1523838/116575060-9935ac80-a90e-11eb-8f93-a732fe5be7b0.png) versus 
![Schermafbeelding 2021-04-29 om 17 16 51](https://user-images.githubusercontent.com/1523838/116575196-b8343e80-a90e-11eb-901a-7cc5bf7e57ce.png)